setup.cfg

```diff
@@ -40,6 +40,8 @@ install_requires =
     astor
     attrs
     docutils
+    sre_parse36
+    typing_extensions; python_version < "3.8"
```

Comment:
It would be best not to have typing_extensions module as a runtime dependency but only for the type checking. This means importing typing_extensions.Final inside a if TYPE_CHECKING block and use string annotation. Pydoctor should support that already.